editor/src/components/editor/editor-component.tsx

```diff
@@ -558,6 +562,7 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
         keyUp={onWindowKeyUp}
       />
       <CommentMaintainer />
+      <ClearSelectionOnNavigation />
```

Comment:
Can this be a hook instead of a component?
```tsx
function useClearSelectionOnNavigation() {
  const dispatch = useDispatch()
  const [remixNavigation] = useAtom(RemixNavigationAtom)
  const paths = Object.values(remixNavigation)
    .map((n) => n?.location.pathname ?? '')
    .join('-')

  React.useEffect(() => {
    dispatch([EditorActions.clearSelection()])
  }, [dispatch, paths])
}
```

Reply:
good idea, pushed it https://github.com/concrete-utopia/utopia/pull/5566/commits/f7497fb27601f2296f270bb8f94a701bd13d517f